application/common/application_data.h

```diff
@@ -158,6 +159,7 @@ class ApplicationData : public base::RefCountedThreadSafe<ApplicationData> {
   bool LoadVersion(base::string16* error);
   bool LoadDescription(base::string16* error);
   bool LoadWindowSetting(base::string16* error);
+  bool LoadDisplayMode(base::string16* error);
```

Comment:
ok, I'm just not happy that ApplicationData is aware of different config file formats, we should have wgt_parser and xpk(manifest)_parser that would initialize a generic ApplicationData object, anyway it's out of this PR scope.

If you keeping the existing design pls use it for wgt as well (remove GetWidgetWindowShowState from Application class )